src/Microsoft.Bcl.AsyncInterfaces/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
Should we give it a unique assembly and package version so that it won't be so unusual if it needs to ship out of sync with the rest of the product?